docs/04-Services/04-Navigation-Manager.md

```diff
@@ -0,0 +1,47 @@
+---
+since: 0.1.1
+---
+
+# Navigation Manager
+
+## Service Overview
+
+This service simplifies keyboard navigation for a set of DOM elements. It exposes a factory function `createNavManager()` that creates an object containing:
+
+- `elementsInDomOrder$` - a store containing the navigable elements in DOM order
+- `directive` - a directive to use on some DOM elements, both to add the keydown event handler and to specify which elements should be managed (either by directly putting the directive on those elements, or by putting the directive on a parent element and specifying which child elements should be included through a `selector` function via `NavManagerItemConfig`)
+- Utilities to navigate the items:
+  - `refreshElements` - method to manually refresh the `elementsInDomOrder$` to capture the latest state of DOM
+  - `focusIndex` - method to focus the element at the given index within `elementsInDomOrder$`
+  - `focusPrevious` - method to focus the previous element, respecting the anscestor direction
+  - `focusNext` - method to focus the next element, respecting the anscestor direction
+  - `focusFirst` - method to focus the first element, respecting the anscestor direction
+  - `focusFirstLeft` - method to focus the element at the left-end of the list
+  - `focusFirstRight` - method to focus the element at the right-end of the list
+  - `focusLast` - method to focus the last element, respecting the anscestor direction
+  - `focusLeft` - method to focus the next focusable element to the left of the currently focused element
+  - `focusRight` - method to focus the next focusable element to the right of the currently focused element
+
+## Usage
+
+Whenever you want to custom keyboard navigation within a defined set of DOM elements you can simply:
+
+- use the factory function to create the NavManager object
+- use the `NavManagerItemConfig` to configure the NavManger by setting:
+  - `keys` - a map of key handlers. The key in the map should match the key event (for example, "ArrowLeft" or "Ctrl+PageDown"). The value in the map is the corresponding key handler. (for example, "FocusNext" or "FocusPrevious")
```

Comment:
```suggestion
  - `keys` - a map of key handlers. The key in the map should match the key event (for example, `"ArrowLeft"` or `"Ctrl+PageDown"`). The value in the map is the corresponding key handler (for example, the `focusNext` or `focusPrevious` function returned by `createNavManager`)
```
(so that people don't think the value is a string)